### PR TITLE
feat: expose more envs in udf and udsink

### LINKED ISSUE #5: expose Namespace, Pipeline and Vertex name as environment variables
As a UDF or UDSink developer, I would like to emit custom metrics. WIth the custom metrics, I would like to attach namespace, vertex, and pipeline names as tags so I can filter those metrics easily. Today there is no way to get these names.

### PR DESCRIPTION
**Pull request by whynowy**
Signed-off-by: Derek Wang <whynowy@gmail.com>

Closes #5.

### CODE REVIEW

File: pkg/isb/jetstream/writer_test.go [hunk 1]
```diff
@@ -165,6 +165,9 @@ func TestJetStreamBufferWriterBufferFull(t *testing.T) {
 	_, errs := jw.Write(ctx, messages)
 	assert.Equal(t, len(errs), 10)
 
+	// wait 2 seconds so that the buffer full check is done.
+	time.Sleep(2 * time.Second)
```
Comment (vigith) [inline pkg/isb/jetstream/writer_test.go:169]: adding this wait won't have any benefit. the assertion has already happened.

Comment (whynowy) [reply pkg/isb/jetstream/writer_test.go:169]: Adding this is to make sure the isFull check has been run.

Comment (whynowy) [reply pkg/isb/jetstream/writer_test.go:169]: Which is for line 172.

Comment (whynowy) [reply pkg/isb/jetstream/writer_test.go]: Updated test case.

**Review by vigith — APPROVED** [review]

**Pull request merged**